pkg/trace/writer/payload.go

```diff
@@ -155,14 +158,34 @@ func (s *queuableSender) Start() {
 // Run executes the queuableSender main logic synchronously.
 func (s *queuableSender) Run() {
 	defer close(s.exit)
+	sema := make(chan struct{}, runtime.NumCPU()*2) // maximum concurrent goroutines flushing
```

Comment:
@djova not sure if this is high enough. Should we maybe use 100? We’ve had successful results with 100. I don’t think this needs to be configurable, only reasonably limited.